scripts/core/register-extensions.tsx

```diff
@@ -5,8 +5,12 @@ import {IExtension, IPage, IWorkspaceMenuItem, IExtensionActivationResult, ISupe
 import {extensions as extensionsWithActivationResult} from 'appConfig';
 import {dispatchInternalEvent} from './internal-events';
 
+interface IExtensionModule {
```

Comment:
The boilerplate in question is 27 chars long `.then((res) => res.default)`. I'd rather keep the interface cleaner and use the boilerplate. It's only needed in the index/config file which is rarely edited anyway.

Reply:
it has to be set in every repo on every branch, that's not so rare imo

Reply:
Maybe you are setting things up more frequently, but even then, it's 10 seconds to type. That's one of the stranger abstractions I've seen, but go ahead if it helps :)

Reply:
:smile: